common/djangoapps/student/signals/signals.py

```diff
@@ -5,7 +5,7 @@
 
 from django.dispatch import Signal
 
-ENROLLMENT_TRACK_UPDATED = Signal(providing_args=['user', 'course_key', 'mode', 'countdown'])
```

Comment:
From [Django 3.1 Release Notes ](https://docs.djangoproject.com/en/3.2/releases/3.1/)
> The purely documentational providing_args argument for Signal is deprecated. If you rely on this argument as documentation, you can move the text to a code comment or docstring.

Reply:
Should we add this info to code comments as the docs suggested, instead of just deleting it?